### PR TITLE
grouping pathological behaviors (interesting discussion)

### PR DESCRIPTION
**Pull request by dosumis**
grouping pathological behaviors #119

### CODE REVIEW

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -10796,6 +10801,7 @@ owl:qualifier some &apos;temporally extended&apos;)</oboInOwl:note>
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/NBO_0000645">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NBO_0000643"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NBO_0010000"/>
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:10804]: Didnt you want to remove the subClassOf behaviour phenotype assertions as well?

Comment (dosumis) [reply src/ontology/nbo-edit.owl:10804]: I think it would be less disruptive of any existing users to keep it.  Where there is a logical def, phenotypes use an EQ pattern, rather than QE as everyone else does, so no formal upper ontology issue I think.

Comment (matentzn) [reply src/ontology/nbo-edit.owl:10804]: Yeah but we will pull in that "behaviour phenotype" class - I was asked to not only remove subclasses of that, but the entire branch, to avoid confusion..

**Comment by dosumis** [comment]
Changed to draft for now as I think it was wrong to put this under GO biological_process, which is intended to group non-pathological processes.  I think ideally we would import MPATH 'pathological process' as a parent class - does that seem reasonable?

**Comment by matentzn** [comment]
Yeah, sounds good I think

File: src/ontology/nbo-edit.owl (hunk 1 above)
Comment (matentzn) [reply src/ontology/nbo-edit.owl:10804]: not just bottom - any module extract that will pull in parents!

**Comment by DitchingIt** [comment]
_TLDR: Almost all current phenotype classes are pathological or redundant. There is potential to reframe a large set of true non-pathological phenotypes._

Looking through the behavioral phenotype subclasses:

1. some are labelled as disorders (i.e. **pathological**) specifically or in their superclass label, e.g. amnestic disorder (NBO:0000269) or dissociative disorders (NBO:0000252)

2. some can only be **pathological**, e.g. delirium (NBO:0000267)

3. some are really only normal variants in a corollary sense (like the rear face of the coin), e.g. we may feel that we experience a degree of non-pathological forgetfulness (NBO:0000606) or clumsiness/dysmetria (NBO:0000592) at times, but the normal variation is really in degrees of recall and dexterity (i.e. forgetfulness and dysmetria are on the **pathological** side of the coin, although the pathology may be quite limited)

4. some effectively remain **redundant** superclasses awaiting class members - including all terminal classes ending in the word 'phenotype', e.g. learning behavior phenotype (NBO:0000766)

5. some are potentially universal, but are none-the-less **pathological**, e.g. senility (NBO:0000572)

6. some might be considered normal variations because they are so common, but are disabling, can cause distress and are potentially amenable to treatment, e.g. acrophobia (NBO:0000310) as a **pathology** (like all true phobias), versus varying degrees of fear of heights not amounting to a phobia

7. some might be signs of a **pathology**, e.g. alcohol preference (NBO:0000140) or obesity (NBO:0000242) as signs of an addiction or endocrine disorder respectively

8. a good number could have an incipient new **phenotype** defined based on them, e.g. fatigue (NBO:0000569) could inspire a new phenotype class along the lines of 'endurance'; similarly with some of the mood disorders ('optimistic', 'pessimistic', etc)

9. a complete list of those that do seem to remain true non-pathological variant **phenotypes** (although some might be better defined) is:
- increased amount of liquid in a single drinking act (NBO:0000851)
- increased amount of liquid in drinking regulation (NBO:0000886)
- alcohol aversion (NBO:0000139)
- food aversion (NBO:0000142)
- saccharin preference (NBO:0000143)
- mating frequency (NBO:0000144)
- mating preference (NBO:0000145)
- asleep (NBO:0000067)
- sleep pattern (NBO:0000158)

ISSUES ARISING:
- [ ] Have I misunderstood issue #120?
- [ ] Aside from phenotypes listed in 9. above (although I am open to argument) I agree that the rest should be assigned to a pathology branch
- [ ] What structure might retain non-pathological phenotypes in the NBO without confusing them with pathologies?
- [ ] How can the true phenotype list be populated? (I am open to offering suggestions based on the current list, although I am not yet capable of drafting changes myself)

**Comment by matentzn** [comment]
@DitchingIt great to see you here as well. What is your interest in NBO?

We are beginning to consider pulling NBO apart a bit. "phenotype" classes really should be moved to [OBA](https://github.com/obophenotype/bio-attribute-ontology/issues/175). Whether a process is pathological will often be a real hazy thing to determine. I personally think we should avoid such judgements as they are really about conventions around "normality", which often is associated with "wild-type" and not really very meaningful for analysis. We only care that "something has changed", and if we want to know the degree of the change, we should say that separately (using terms like pathological or severe or some such are ok then, but I would kinda like to avoid classes like this here).

The actual "pathological phenotypes" should all be in HPO rather than here.

**Comment by pmidford** [comment]
I like where this is going - when we were working on integrating the Animal Behavior Ontology with NBO, we weren't sure what to do with the phenotype terms.  If I'm not mistaken, the formal definition of behavior phenotype in NBO is just a quality of a behavior process, though a more through definition was proposed in at least one publication.  As I recall, there was no relation in the ontology between a phenotype and its process, which seemed to defeat any attempt to reason with them. I haven't found these terms useful in my modeling of spider behavior.  

It would also be nice to see the species specific terms moved to HPO or the appropriate MOD ontology, but I guess that's another issue.

**Comment by DitchingIt** [comment]
> @DitchingIt great to see you here as well. What is your interest in NBO?

Thanks @matentzn 

I was inspired by the publication of the comprehensive Elephant Ethogram last year and how it lets amateurs on safari watch, film, and understand what they see much better. I am interested in the overlap between tropical reef ecology and hobbyist fish watchers and would love to see something akin to it. I couldn't find what I was looking for elsewhere re: fishes, and NBO seems the ideal place for anchoring it, but NBO needs more work to make it serve the purpose, so I am hoping to help enhance the NBO as a stepping stone.

So long as the behavioural processes branch contains all necessary behaviours (one can simply ignore redundant ones like 'submissive behavior towards male mice'), specific traits and variant phenotypes aren't necessary to me. And specifically, I'm not looking for pathological phenotypes, let alone the vast range of experimentally induced ones listed in the Zebrafish Behavior Catalog; very few functionally impaired fishes survive very long in the wild anyway.

**Comment by DitchingIt** [comment]
> Whether a process is pathological will often be a real hazy thing to determine. @matentzn 

I absolutely agree with you regarding judgements. What I was trying to point out was that:
- most of the phenotype branch could be put in a pathology-type ontology, not just the seizures which inspired #120 back in May 2022 (you have mentioned HPO)
- a few items in the phenotype branch were not defined as pathological (the ones I bulleted in my point 9), and many of the others could inspire definitions which express variation rather than pathology, which may be more satisfying e.g. to modelling spider behaviour (anyway you suggest OBA is where they belong and I imagine that will be where this kind of discussion would in future progress)

My main thrust with my comment was to encourage progress on this issue and if the process branch is all that is left in the NBO, I think it will allow greater focus; really I am only interested in the process branch but as a newbie, I didn't know if I could say, "the phenotype branch looks like a confusing red herring to me".

Incidentally, is there a way to know how each class in NBO features in other ontologies, and in particular the phenotypes? If @pmidford doesn't use them, who does? @dosumis thought it might be disruptive to other users to remove phenotypes from the ontology.

**Comment by pmidford** [comment]
Yes, several of us would like to see NBO evolve into something suitable 
as a basis for ethograms.  Glad there are others.

**Comment by pmidford** [comment]
I am not using anything from the phenotypes branch, nor do I plan to.    Have we heard anything from George, Robert or Paul on these changes?  Or has the ownership issue been tacitly resolved?  I don't have any objections to this split, but after all the effort that @aclark-binghamton-edu and others put into integrating material from the ABO ontology, was the solution really just to wait it out?

**Comment by DitchingIt** [comment]
> I think the truth is, we need some kind of grant for behaviour modelling.. Before that, we are at the mercy of people like @DitchingIt to incorporate the ABO terms, or doing any major restructuring.
> 
> The phenotypes branch would not be deleted, just to be clear, just moved to a phenotype ontology (probably uPheno).

So axiomatically speaking, if I'm given a grant, you are not at my mercy... ;-)
